scalatest/src/main/scala/org/scalatest/exceptions/StackDepthExceptionHelper.scala

```diff
@@ -152,7 +152,7 @@ private[scalatest] object StackDepthExceptionHelper extends Serializable {
   }
 
   def getStackDepthFun(pos: source.Position): (StackDepthException => Int) = { sde =>
-    getStackDepth(sde.getStackTrace, pos)
+    getStackDepth(sde.getStackTrace.map(_.nn), pos)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.